### PR TITLE
Create user registration component

### PR DESCRIPTION
**Pull request by wmemorgan**
https://trello.com/c/pg7Ifz7N
https://trello.com/c/vyV57ifN

### CODE REVIEW

**Comment by wmemorgan** [comment]
[Create user registration page](https://trello.com/c/pg7Ifz7N/62-create-user-registration-page)

File: socialtours/src/components/Register.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	clientID: "mKqnZoQovxuLSlTUSIwjj4bcuMOH3aX1"
+});
+
+class Register extends Component {
+	state = {
+		email: "",
+		first_name: "",
+		last_name: "",
+		password: "",
+		phone: "",
```
Comment (bcuz) [inline socialtours/src/components/Register.js]: doesnt prevent it from registering, but i dont see a phone column in the users table

**Comment by bcuz** [comment]
registration stuff looks good to me. not much need to build off of my unmerged commits

File: socialtours/src/components/Register.js (hunk 1 above)
Comment (wmemorgan) [reply socialtours/src/components/Register.js]: I updated the database schema to include a `phone_nbr` field. The data model documentation has been updated with the change:

https://dbdiagram.io/d/5d4b8094ced98361d6dd6837

**Comment by wmemorgan** [comment]
> registration stuff looks good to me. not much need here to build off of my unmerged commits

Thanks. Today I'm going to port the registration methods over to Redux. I wanted to make sure the basic code logic works.

**Comment by wmemorgan** [comment]
@bcuz @RaneWallin - the migration to Redux is complete and the branch is ready to go. Review and merge at your earliest convenience.

File: socialtours/src/components/Register.js (hunk 1 above)
Comment (bcuz) [reply socialtours/src/components/Register.js]: should there be a migration in the backend repo for phone_nbr?

Comment (wmemorgan) [reply socialtours/src/components/Register.js]: @bcruz - I have a phone_nbr migration created will merge the branch to master on my 5th day (Sunday night).

**Pull request merged**